src/workspaces/features/RunPipelineDialog/RunPipelineDialog.tsx

```diff
@@ -88,7 +88,7 @@ const RunPipelineDialog = (props: RunPipelineDialogProps) => {
       const errors = {} as any;
       const { version, ...fields } = values;
       version?.parameters
-        .filter((param) => param.type !== "bool")
+        .filter((param) => param.type !== "bool" || !param.required)
```

Comment:
In fact we could have a boolean that is required to be true (like a check that the user agrees to something).

I would do a first check to be sure the version is there (or put an error as well) and then check for all required parameters like you do.

Your check on `!fields[param.code] is not correct since 0 is a value as well`

Reply:
I don't get the part about _check if the version is there_.

Reply:
Since it's not possible to submit the form without having a version, why not checking the value of the version as well ?

Reply:
I'm afraid to not understand it.